### PR TITLE
Pm 2020 11 module detail clean

### PR DESCRIPTION
**Pull request by phillimorland**
only way to make it work with designs was to add another template

### CODE REVIEW

**Review by fuzzylogic2000 — COMMENTED** [review]
It's a good idea and looks good!
But currently, the button is there and I can go to the new view when the phase is not active. What do we currently do about that? We have some info on the page when it starts or that participation is not possible at the moment? Maybe we can merge this and then add some info/behaviour for that next?

**Comment by phillimorland** [comment]
@fuzzylogic2000 shall I just hide the button when phase not active?

**Comment by fuzzylogic2000** [comment]
@phillimorland That wouldn't work for after the phase, because people are still meant to view the results, right? Maybe don't display the react before the phase and instead The sentence "participation is not possible at the moment, it starts at..."

File: adhocracy-plus/templates/a4modules/module_detail.html [hunk 1]
```diff
@@ -59,22 +48,6 @@
             <div class="container">
                 <div class="row mb-5">
                     <div class="col-md-8 col-lg-6 offset-md-2 offset-lg-3">
-                        {% with container=project.containers.first %}
-                            {% if container %}
-                                <a href="{{ container.get_absolute_url }}" class="project-header__link" data-embed-target="external">
-                                    <i class="fa fa-arrow-left" aria-hidden="true"></i>
-                                    {% blocktrans with name=container.name %}to project collection {{ name }}{% endblocktrans %}
```
Comment (rmader) [inline adhocracy-plus/templates/a4modules/module_detail.html:66]: Do we want to remove that for all modules?

File: apps/questions/templates/a4_candy_questions/question_module_detail.html [hunk 2]
```diff
@@ -17,9 +13,10 @@ <h3>{{ module.name }}</h3>
                 <div class="mb-3">
                     {{ module.description | transform_collapsibles | richtext }}
                 </div>
-                <a class="btn btn--primary" href="{% url 'module-detail' module.project.organisation.slug module.slug %}">{% trans 'Go to event' %}</a>
+                {% if not module.future_phases %}
```
Comment (phillimorland) [inline apps/questions/templates/a4_candy_questions/question_module_detail.html:16]: @fuzzylogic2000 this seemed like a better choice as we don't repeat ourselves now

File: adhocracy-plus/templates/a4modules/module_detail.html (hunk 1 above)
Comment (phillimorland) [reply adhocracy-plus/templates/a4modules/module_detail.html:66]: I didn't find any trace of project containers being used as it seemed like a left over from mB so thought I would clean it up

File: apps/questions/templates/a4_candy_questions/question_module_detail.html (hunk 2 above)
Comment (fuzzylogic2000) [reply apps/questions/templates/a4_candy_questions/question_module_detail.html:16]: Yes, you're right!

File: adhocracy-plus/templates/a4modules/module_detail.html (hunk 1 above)
Comment (rmader) [reply adhocracy-plus/templates/a4modules/module_detail.html:66]: Ah right

**Comment by rmader** [comment]
Changes included in https://github.com/liqd/adhocracy-plus/pull/962

**Pull request closed without merging**